packages/x-charts-pro/src/FunnelChart/funnelSectionClasses.ts

```diff
@@ -28,10 +29,10 @@ function getFunnelSectionUtilityClass(slot: string) {
 }
 
 export const useUtilityClasses = (props: FunnelSectionProps) => {
-  const { classes, seriesId, variant } = props;
+  const { classes, seriesId, variant, dataIndex } = props;
 
   const slots = {
-    root: ['root', `series-${seriesId}`],
+    root: ['root', `series-${seriesId}`, `data-index-${dataIndex}`],
```

Comment:
I thought we were going to use `data` attributes for these values instead of classes, similar to what Alex is doing [here](https://github.com/mui/mui-x/pull/18048). 

We can't remove the current `series-${seriesId}` without a breaking change, but I think we can add `data-series` and `data-index` already, right?

Reply:
true, I put them in just to be in line with the current API.

Will add the new attributes